core/Test/Framework/Runners/Console.hs

```diff
@@ -24,16 +22,7 @@ import System.Exit
 import System.IO
 
 import Data.Monoid
-
-#if !MIN_VERSION_base(4,7,0)
-instance Functor OptDescr where
-    fmap f (Option a b arg_descr c) = Option a b (fmap f arg_descr) c
-
-instance Functor ArgDescr where
-    fmap f (NoArg a) = NoArg (f a)
-    fmap f (ReqArg g s) = ReqArg (f . g) s
-    fmap f (OptArg g s) = OptArg (f . g) s
-#endif
+import Data.Orphans ()
```

Comment:
I can appreciated that, but we don't want to depend on `base-orphans` when it's not necessary. And in fact we only need it for older GHCs :-/